### PR TITLE
PAYARA-3256 Command Threadpool uses the underlying Payara Executor Service

### PR DESCRIPTION
**Pull request by smillidge**
As part of the reduction of thread usage task. Refactor the admin command thread pool to use the Payara Executor Service

### CODE REVIEW

**Comment by arjantijms** [comment]
Jenkins test please

**Comment by arjantijms** [comment]
Jenkins test please

**Comment by Pandrex247** [comment]
Jenkins test please

File: nucleus/admin/util/src/main/java/com/sun/enterprise/admin/util/CommandThreadPool.java [hunk 1]
```diff
@@ -82,48 +83,19 @@
     @Inject
     private Logger logger;
 
-    private ExecutorService svc = null;
+    @Inject
+    PayaraExecutorService executor;
```
Comment (svendiedrichsen) [inline nucleus/admin/util/src/main/java/com/sun/enterprise/admin/util/CommandThreadPool.java]: Missing `private` qualifier

**Comment by smillidge** [comment]
Yeah the fix is in a different PR I just need to figure out the order

**Comment by Pandrex247** [comment]
Jenkins test please

**Comment by smillidge** [comment]
Needs https://github.com/payara/Payara/pull/3371 merging first

**Comment by Pandrex247** [comment]
Jenkins test please

**Comment by Pandrex247** [comment]
Jenkins test please

**Comment by arjantijms** [comment]
>Needs #3371 merging first

Had been merged now

**Comment by smillidge** [comment]
jenkins test please

**Comment by smillidge** [comment]
jenkins test please

**Comment by smillidge** [comment]
Closing this while I work out what is wrong with it

**Pull request closed without merging**